doc/examples/effects/simple/pubspec.yaml

```diff
@@ -10,4 +10,4 @@ dependencies:
   flutter:
```

Comment:
Didn't the `doc` folder was renamed to `docs`?

Reply:
Haha, I have a slight memory of someone doing that change. ;)
Have to rebase the branch.

Reply:
Rebased!